test/torchaudio_unittest/functional/functional_cpu_test.py

```diff
@@ -179,3 +191,31 @@ def test_mask_along_axis_iid(self, mask_param, mask_value, axis):
 
         assert mask_specgrams.size() == specgrams.size()
         assert (num_masked_columns < mask_param).sum() == num_masked_columns.numel()
+
+
+@skipIfNoExec('sox')
```

Comment:
`sox` command should not be required. This is a smoke test that does not require any external tool.